test/parallel/test-http-server-keep-alive-timeout.js

```diff
@@ -1,77 +1,122 @@
 'use strict';
 
-const common = require('../common');
-const assert = require('assert');
-const http = require('http');
-const net = require('net');
+module.exports.testServerKeepAliveTimeout = function(common, isHttps) {
```

Comment:
As per https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md
put `const common = require('../common');` as L2 then a blank line and then a comment 
```js
// Testing http[s] server keepalive semantics.
// Reusing tests with test-https-server-keep-alive-timeout.js
// Ref: https://github.com/nodejs/node/pull/13448
```